drltrace/drltrace_frontend.cpp

```diff
@@ -142,22 +155,34 @@ print_version() {
 
 static void
 configure_application(char *app_name, char **app_argv, void **inject_data,
-                      const char *dr_root, const char *lib_path)
+                      const char *dr_root, const char *lib_path, const char *log_dir)
 {
     bool is_debug = false;
 #ifdef DEBUG
     is_debug = true;
 #endif
     int errcode;
+    drfront_status_t sc;
+    ssize_t len;
+    size_t sofar = 0;
     char *process;
     process_id_t pid;
     char dr_option[MAX_DR_CMDLINE];
+    char drltrace_option[MAX_DR_CMDLINE];
     dr_option[0] = '\0';
 
     if (op_follow_children.get_value() == false)
         dr_snprintf(dr_option, BUFFER_SIZE_ELEMENTS(dr_option), "-no_follow_children");
     NULL_TERMINATE_BUFFER(dr_option);
 
+    BUFPRINT(drltrace_option, BUFFER_SIZE_ELEMENTS(drltrace_option), sofar, len, "%s ",
```

Comment:
we have warning here deprecated conversion from string constant to 'char*'.  I will prepare a pull request changing drfront_bufprint's fmt from char* to const char *